src/applications/facility-locator/config.js

```diff
@@ -33,9 +35,64 @@ export const api = {
  */
 export const ccLocatorEnabled = () => true;
 
+/**
+ * Build parameters and URL for facilities API calls
+ *
+ */
+export const resolveParamsWithUrl = (
+  address,
+  locationType,
+  serviceType,
+  page,
+  bounds,
+) => {
+  const filterableLocations = ['health', 'benefits', 'cc_provider'];
+  let facility;
+  let service;
+  let url;
+  switch (locationType) {
+    case 'urgent_care':
+      if (!serviceType || serviceType === 'UrgentCare') {
```

Comment:
This is either a 'NonVAUrgentCare' or ('UrgentCare' or nil) you can do this with an if else, if non else everything else.

Reply:
Yeah, done